src/OrchardCore/OrchardCore.Abstractions/Shell/Configuration/ShellConfiguration.cs

```diff
@@ -164,5 +166,25 @@ public IChangeToken GetReloadToken()
         {
             return Configuration.GetReloadToken();
         }
+
+        protected virtual void Dispose(bool disposing)
+        {
+            if (!disposedValue)
```

Comment:
Cool!

Can we name it `_disposed`, at least with a leading underscore as it is a private field?

Reply:
Yeh - This was the value that VS gave when you do Alt+Enter.